src/xcode/ENA/ENA/Source/AppDelegate & Globals/ENATaskExecutionDelegate.swift

```diff
@@ -120,16 +125,16 @@ class TaskExecutionHandler: ENATaskExecutionDelegate {
 		service.submitExposure { error in
 			switch error {
 			case .noSubmissionConsent:
-				self.updateStoreWithKeySubmissionInBackground(value: false)
+				Analytics.log(.keySubmissionMetadata(.submittedInBackground(false)))
```

Comment:
I would rename log to collect or store as the word "log" is misleading in this context.